### PR TITLE
feat(theme-designer): init color preview

### PR DESCRIPTION
**Pull request by m0ksem**
Fix va-config colors prop
Fix ColorInput perfomance
Add simple theme designer

image

### CODE REVIEW

File: packages/docs/page-config/styles/colors/components/ThemeDesigner.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<script setup lang="ts">
```
[the remark this answers is not in the document]
Comment (m0ksem) [reply packages/docs/page-config/styles/colors/components/ThemeDesigner.vue:1]: Nope. It is a docs page, not a component, so it doesn't have a story.

**Comment by m0ksem** [comment]
I'll take this PR as MVP of the theme designer. It is enough for now, but we definitely improve it later.

**Comment by asvae** [comment]
Sergey asked for this feature.

**Review by asvae — APPROVED** [review]
🤟

**Pull request merged**